### PR TITLE
[EDU-3426] chore: final uxw check

### PR DESCRIPTION
**Pull request by bru-andrade**
### What is the new behavior introduced by this PR?
Modifies labels, titles, breadcrumbs, and helper texts to maintain consistency in writing.

[Full list of changes](https://aziontech.slack.com/archives/C04C9HKEN9F/p1710259478021229?thread_ts=1710259467.736819&cid=C04C9HKEN9F)

### Does this PR introduce breaking changes?
- [x] No
- [ ] Yes 

### Does this PR introduce UI changes? Add a video or screenshots here.
Too many to add screenshots.

### Does it have a link on Figma?

<hr />

#### Make sure your pull request fits the checklist below (when applicable):

- [x] The issue title follows the format: [ISSUE_CODE] TYPE: TITLE
- [x] Commits are tagged with the right word (fix, feat, test, etc)
- [ ] User inputs are sanitized/protected against malicious attacks
- [x] Tags are added to the PR

#### These changes were tested on the following browsers:
- [x] Chrome
- [ ] Edge
- [ ] Firefox
- [ ] Safari

### CODE REVIEW

**Comment by bru-andrade** [comment]
Opened as draft for validation from the writers.

**Review by gabriel-azion — APPROVED** [review]

File: src/services/metrics-services/constants/dashboards-texts.js [hunk 1]
```diff
@@ -103,7 +103,7 @@ const DASHBOARDS_TEXTS = {
     waf: {
       threats_vs_requests: {
         description:
-          'Sum of attacks processed by WAF. Displays the total amount of Waf Requests Blocked, Waf Requests Threats, and Waf Requests Allowed.'
+          'Sum of attacks processed by WAF. Displays the total amount of WAF Requests Blocked, WAF Requests Threats, and WAF Requests Allowed.'
```
Comment (hannah-sk) [inline src/services/metrics-services/constants/dashboards-texts.js:106]:
```suggestion
          'Sum of attacks processed by WAF. Displays the total amount of Waf Requests Blocked, Waf Requests Threats, and Waf Requests Allowed.'
```

Comment (hannah-sk) [reply src/services/metrics-services/constants/dashboards-texts.js:106]: refletindo a legenda.

File: src/views/Credentials/FormFields/FormFieldsCreateCredentials.vue [hunk 2]
```diff
@@ -70,15 +70,15 @@
           class="w-full"
           :disabled="props.generatedToken"
         />
-        <small class="text-color-secondary text-xs font-normal leading-5">
-          Description of credential, e.g.: Credential used for clients XYZ.
+        <small class="text-xs text-color-secondary font-normal leading-5">
+          Description and purpose of credential.
```
Comment (hannah-sk) [inline src/views/Credentials/FormFields/FormFieldsCreateCredentials.vue:74]:
```suggestion
          Description and purpose of the credential.
```

File: src/views/PersonalTokens/FormFields/FormFieldsPersonalToken.vue [hunk 3]
```diff
@@ -158,13 +160,16 @@
...
+          Include a description to specify the token's purpose or usage.
+        </small>
       </div>
     </template>
   </FormHorizontal>
 
   <FormHorizontal
     title="Token"
-    description="Define the token expiration date by selecting one of the suggested date ranges. For security matters, you can only copy the Personal Token right after you create it. In case you need the Personal Token code after that, you must create a new one."
+    description="Define the token expiration date by selecting one of the suggested date ranges. For security matters, the personal token will only be available right after it is created."
```
Comment (hannah-sk) [inline src/views/PersonalTokens/FormFields/FormFieldsPersonalToken.vue:172]:
```suggestion
    description="Define the token expiration date by selecting one of the suggested date ranges. For security matters, the personal token will only be available right after it's created."
```

File: src/views/WafRules/ListWafRulesTuning.vue [hunk 4]
```diff
@@ -38,15 +38,15 @@
...
+    pageTitleDelete="WAF rules tuning"
     :columns="wafRulesAllowedColumns"
     :hasListService="true"
     :dataFilted="dataFiltedComputed"
     @on-select-data="selectedItems"
     :cleanSelectData="cleanSelectData"
     :showselectionMode="true"
     :editInDrawer="openMoreDetails"
-    emptyListMessage="No WAF Rules Tuning found."
+    emptyListMessage="No WAF rules tuning found."
```
Comment (hannah-sk) [inline src/views/WafRules/ListWafRulesTuning.vue:49]:
```suggestion
    emptyListMessage="No requests found."
```

File: src/views/DigitalCertificates/ListView.vue [hunk 5]
```diff
@@ -9,19 +9,19 @@
...
         :columns="getColumns"
-        pageTitleDelete="Digital Certificate"
+        pageTitleDelete="digital certificate"
         editPagePath="digital-certificates/edit"
-        addButtonLabel="Digital Certificates"
+        addButtonLabel="Digital Certificate"
         createPagePath="digital-certificates/create"
         @on-load-data="handleLoadData"
-        emptyListMessage="No Digital Certificate found."
+        emptyListMessage="No digital certificates found."
```
Comment (MarianaAguilera) [inline src/views/DigitalCertificates/ListView.vue:17]:
```suggestion
        emptyListMessage="No digital certificates found"
```

File: src/views/DigitalCertificates/ListView.vue [hunk 6]
```diff
@@ -9,19 +9,19 @@
...
       />
 
       <EmptyResultsBlock
         v-else
-        title="No digital certificates added"
-        description="Create your first digital certificates."
-        createButtonLabel="Digital Certificates"
+        title="No digital certificates have been added"
+        description="Click the button below to add your first digital certificates."
```
Comment (MarianaAguilera) [inline src/views/DigitalCertificates/ListView.vue:23]:
```suggestion
        description="Click the button below to add your first digital certificate."
```

File: src/views/EdgeApplicationsOrigins/FormFields/FormFieldsEdgeApplicationsOrigins.vue [hunk 7]
```diff
@@ -447,8 +460,8 @@
           icon="pi pi-plus-circle"
           outlined
           type="button"
-          aria-label="New Origin"
-          label="New Origin"
+          aria-label="Add Origin"
+          label="Add Origin"
```
Comment (MarianaAguilera) [inline src/views/EdgeApplicationsOrigins/FormFields/FormFieldsEdgeApplicationsOrigins.vue:464]: aqui deveria ser só `Origin`?

Comment (bru-andrade) [reply src/views/EdgeApplicationsOrigins/FormFields/FormFieldsEdgeApplicationsOrigins.vue:464]: Boa pergunta, na tela fica assim:

image

Mas pelo padrão seguido em outras telas:

image

image

Só na Rules Engine fica diferente

image

Comment (MarianaAguilera) [reply src/views/EdgeApplicationsOrigins/FormFields/FormFieldsEdgeApplicationsOrigins.vue:464]: valeu, tá ok então para manter o padrão com as outras, sim

**Review by MarianaAguilera — DISMISSED** [review]

**Review by hannah-sk — APPROVED** [review]

**Review by MarianaAguilera — APPROVED** [review]

File: src/views/RealTimePurge/FormFields/FormFieldsCreateRealTimePurge.vue [hunk 8]
```diff
@@ -12,12 +12,12 @@
 
   const computedPurgeArgumentsPlaceHolder = computed(() => {
     if (purgeType.value === 'cachekey') {
-      return 'https://www.example.com.br/index.html'
+      return 'httpswww.example.com/images/image.jpg'
```
Comment (william-tome) [inline src/views/RealTimePurge/FormFields/FormFieldsCreateRealTimePurge.vue:15]:
```suggestion
      return 'https://www.example.com/images/image.jpg'
```

File: src/views/PersonalTokens/FormFields/FormFieldsPersonalToken.vue [hunk 9]
```diff
@@ -102,7 +102,7 @@
 <template>
   <FormHorizontal
     title="General"
-    description="Choose a descriptive and easy to remember name. Include a description to specify the token's purpose or usage."
+    description=""
```
Comment (william-tome) [inline src/views/PersonalTokens/FormFields/FormFieldsPersonalToken.vue:105]:
```suggestion
```

**Review by lucasmendes21 — APPROVED** [review]

File: src/views/RealTimePurge/FormFields/FormFieldsCreateRealTimePurge.vue (hunk 8 above)
Comment (lucasmendes21) [reply src/views/RealTimePurge/FormFields/FormFieldsCreateRealTimePurge.vue:15]: não tinha visto.

Comment (william-tome) [reply src/views/RealTimePurge/FormFields/FormFieldsCreateRealTimePurge.vue:15]: Não é um erro. A Bruna me explicou aqui

**Pull request merged**